contents/handbook/company/employee-expectations.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Expectations for PostHog employees
+sidebar: Handbook
+showTitle: true
+---
+
+## Introduction
+
+People who work at PostHog have come from all sorts of different backgrounds - large companies, small companies, agencies, single-founder startups. The modes of working at these places necessarily differ from each other and from PostHog. However, when you're brought on to work as an employee at PostHog, we expect you to work in some fairly specific ways. And being the transparent company that we are, we want to let you know about those expectations, because you can only meet expectations when you are aware of them.
+
+Generally, our [values](https://posthog.com/handbook/values) are a great place to start, as well as our doc on [culture](https://posthog.com/handbook/company/culture). Here are a few specific ways to apply those values and cultures:
+
+## Ask for help, but only after you've tried first
+
+If you've just joined us, there's a lot you probably don't know. That's okay! However, we _do_ expect that you try to help yourself before asking for help. Here's a framework to use as a guide:
+
+- If it's an internal product
+  - We expect you to read our docs
+  - We expect you to search through slack
+  - If it's in beta, it might be buggy or missing features - definitely request/report them, but also figure out how to do your job with plan B and plan C if it won't be fixed quickly -- i think this is what we generally call grit
```

Comment:
```suggestion
  - If it's in beta, it might be buggy or missing features - definitely request/report them, but also figure out how to do your job with plan B and plan C if it won't be fixed quickly – I think this is what we generally call grit
```